### PR TITLE
Include ICS Calendar Invites to Event Signup Acceptance Emails

### PR DESCRIPTION
**Pull request by kunerj**
It would be nice to have calendar invites for events to lower the amount of spontaneous de-registrations because of people forgetting about events.

### CODE REVIEW

**Review by temparus — COMMENTED** [review]
Thanks for your contribution! This is a great addition to the application.

Overall the changes look good. I have suggested some minor improvements directly in the code.

**Comment by kunerj** [comment]
Thanks for your valuable comments! I implemented your suggested changes.

File: amivapi/templates/events_accept_calendar_invite.ics [hunk 1]
```diff
@@ -5,7 +5,6 @@ X-MS-OLK-FORCEINSPECTOROPEN:TRUE
 METHOD:PUBLISH
 BEGIN:VEVENT
 UID:{{ event_id }}@amiv.ethz.ch
-ORGANIZER:MAILTO:{{ reply_to_email }}
```
Comment (temparus) [inline amivapi/templates/events_accept_calendar_invite.ics]: If a client does not support the organizer field, wouldn't it just be ignored?

Comment (suisseWalter) [reply amivapi/templates/events_accept_calendar_invite.ics]: should this now be done this way?

Comment (suisseWalter) [reply amivapi/templates/events_accept_calendar_invite.ics]: apperently no client supports it, and outlook has active problems with it. (some formating breaks for the entire email). therefore it's better to leave it away.

**Review by suisseWalter — APPROVED** [review]
looks good to me. 
most parts of the code seem to not affect any other parts. and it's a usefull feature

**Pull request merged**